app/models/publication.rb

```diff
@@ -207,17 +207,6 @@ def rebuild_authorship
     set_last_updated_value_in_hash
   end
 
-  def add_any_pubmed_data_to_hash
-    return if pmid.blank?
-    pubmed_hash = PubmedSourceRecord.get_pubmed_hash_for_pmid(pmid)
-    return if pubmed_hash.nil?
-
-    pub_hash[:mesh_headings] = pubmed_hash[:mesh_headings] if pubmed_hash[:mesh_headings].present?
-    pub_hash[:abstract] = pubmed_hash[:abstract] if pubmed_hash[:abstract].present?
-    pmc_id = pubmed_hash[:identifier].detect { |id| id[:type] == 'pmc' }
-    pub_hash[:identifier] << pmc_id if pmc_id
-  end
-
```

Comment:
On master
```
$ git grep 'add_any_pubmed_data_to_hash'
app/models/publication.rb:137:      add_any_pubmed_data_to_hash
app/models/publication.rb:292:  def add_any_pubmed_data_to_hash
lib/tasks/cap_cutover.rake:497:      pub.add_any_pubmed_data_to_hash
lib/tasks/cap_cutover.rake:556:      pub.add_any_pubmed_data_to_hash
spec/models/publication_spec.rb:196:  describe 'add_any_pubmed_data_to_hash' do
spec/models/publication_spec.rb:201:      publication.add_any_pubmed_data_to_hash
spec/models/publication_spec.rb:208:      publication.add_any_pubmed_data_to_hash
spec/models/publication_spec.rb:215:      publication.add_any_pubmed_data_to_hash
spec/models/publication_spec.rb:224:      publication.add_any_pubmed_data_to_hash
spec/models/publication_spec.rb:234:      publication.add_any_pubmed_data_to_hash
```

Reply:
Noted in https://github.com/sul-dlss/sul_pub/issues/423 to come back to this later.